splitio/client/factory.py

```diff
@@ -403,7 +455,12 @@ def _build_redis_factory(api_key, cfg):
         'segments': RedisSegmentStorage(redis_adapter),
         'impressions': RedisImpressionsStorage(redis_adapter, sdk_metadata),
         'events': RedisEventsStorage(redis_adapter, sdk_metadata),
+        'telemetry': RedisTelemetryStorage(redis_adapter, sdk_metadata)
     }
+    telemetry_producer = TelemetryStorageProducer(storages['telemetry'])
+    telemetry_consumer = TelemetryStorageConsumer(storages['telemetry'])
```

Comment:
not needed

Reply:
Removed consumer for redis and localhost